programs/sbf/rust/account_mem_deprecated/Cargo.toml

```diff
@@ -9,7 +9,12 @@ license = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
+solana-account-info = { workspace = true }
 solana-program = { workspace = true }
+solana-program-entrypoint = { workspace = true }
```

Comment:
I see you already brought up `solana_program::entrypoint_deprecated!` in Discord so `solana-program` has to stay here for now